lib/browser.ts

```diff
@@ -938,6 +943,9 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * AngularJS.
    */
   getLocationAbsUrl(): wdpromise.Promise<any> {
+    logger.warn(
+        'You are using `browser.getLocationAbsUrl()`, with is deprecated, inaccurately named, ' +
```

Comment:
TMI, just "`browser.getLocationAbsUrl() is deprecated, please use `browser.getCurrentUrl` instead."